.github/workflows/watch-conda-forge.yml

```diff
@@ -0,0 +1,104 @@
+# Check for new versions of dask on Conda Forge
+name: WatchCondaForge
+
+on:
+  schedule:
+    - cron: "0 * * * *"
+
+jobs:
+  check-version:
+    runs-on: ubuntu-latest
+    if: github.repository == 'conda-forge/pangeo-dask-feedstock'
+
+    steps:
+      - name: Checkout Repository
+        uses: actions/checkout@v2
+
+      - name: calver
+        id: calver
+        run: echo "::set-output name=version::$(date +'%Y.%m.%d')"
+
+      - name: Get latest dask version
+        id: latest_version_dask
+        uses: jhamman/gha-anaconda-package-version@feature/calver
+        with:
+          org: "conda-forge"
+          package: "dask"
+
+      - name: Find and Replace Dask
+        id: dask
+        uses: jacobtomlinson/gha-find-replace@0.1.1
+        with:
+          include: "meta.yml"
+          find: "dask =.*"
+          replace: "dask =${{ steps.latest_version_dask.outputs.version }}"
+
+      - name: Get latest distributed version
+        id: latest_version_distributed
+        uses: jhamman/gha-anaconda-package-version@feature/calver
+        with:
+          org: "conda-forge"
+          package: "distributed"
+
+      - name: Find and Replace Distributed
+        id: distriubted
+        uses: jacobtomlinson/gha-find-replace@0.1.1
+        with:
+          include: "meta.yml"
+          find: "distributed =.*"
+          replace: "distributed =${{ steps.latest_version_distributed.outputs.version }}"
+
+      - name: Get latest dask-kubernetes version
+        id: latest_version_k8s
+        uses: jhamman/gha-anaconda-package-version@feature/calver
+        with:
+          org: "conda-forge"
+          package: "dask-kubernetes"
+
+      - name: Find and Replace Dask-kubernetes
+        id: kubernetes
+        uses: jacobtomlinson/gha-find-replace@0.1.1
+        with:
+          include: "meta.yml"
+          find: "dask-kubernetes =.*"
+          replace: "dask-kubernetes =${{ steps.latest_version_k8s.outputs.version }}"
+
+      - name: Get latest dask-gateway version
+        id: latest_version_gateway
+        uses: jhamman/gha-anaconda-package-version@feature/calver
+        with:
+          org: "conda-forge"
+          package: "dask-gateway"
+
+      - name: Find and Replace Dask-Gateway
+        id: gateway
+        uses: jacobtomlinson/gha-find-replace@0.1.1
+        with:
+          include: "meta.yml"
+          find: "dask-gateway =.*"
+          replace: "dask-gateway =${{ steps.latest_version_gateway.outputs.version }}"
+
+      - name: Find and Replace CalVer
+        if: steps.dask.outputs.modifiedFiles > 0 || steps.distributed.outputs.modifiedFiles > 0 || steps.kubernetes.outputs.modifiedFiles > 0 || steps.gateway.outputs.modifiedFiles > 0
```

Comment:
@scottyhq - does this look right to you?

Reply:
seems ok to me